Basic-Recruit-Training/Person.cpp

```diff
@@ -0,0 +1,73 @@
+#include <iostream> //Imports cout
+#include <sstream>
```

Comment:
we should remove librarys that we do not use, sstream shouldnt be needed for this project :)